app/generator/print-styles.js

```diff
@@ -0,0 +1,93 @@
+const { millimetresToPoints } = require('./conversion')
+
+const zeroMillimeters = 0
```

Comment:
you're not going to be happy about this one - can we make it just something like zeroMargin to more clearly represent what this is used for?

Reply:
also done